docs/agents/spec-process.md

```diff
@@ -0,0 +1,69 @@
+Every change that affects more than one agent should be initiated via a change to the specification rather than creating a cross-agent issue.
+
+## Process
+
+1. Agents discussion issue \
+  If there's a lot of uncertainty about what the proposal is,
+  a discussion issue should be created.
+  In doubt, lean towards not creating a discussion issue and start with a draft PR.
+  Discussion issues should not have votes/agent checkboxes.
+  It's not expected that all agents participate in the initial discussions, although everyone is invited to do so.
```

Comment:
I'd highlight sooner that a draft PR is the preferred first step.  Something like:
```
Open a draft PR to change the specification to initiate a discussion.  If discussion is required before a spec change proposal can even be assembled, create an Agent discussion issue first ...
```

Reply:
done. better now?